packages/lit-html/src/hydrate.ts

```diff
@@ -382,7 +375,7 @@ const createAttributeParts = (
         // others to avoid perf impact of touching the DOM unnecessarily
         const noCommit = !(
           instancePart instanceof EventPart ||
-          instancePart instanceof PropertyPart
+          (instancePart as unknown) instanceof PropertyPart
```

Comment:
I'm surprised we're using `instanceof` here. We should be checking `instancePart.type`. @kevinpschaaf

Reply:
Updated, PTAL

Reply:
I think the assumption was that since hydrate is the one making the parts, it's safe to use instanceof. However, compiled template results potentially changes that (ctor might come form another version), so definitely a good change to make.